src/components/button/examples/button.tsx

```diff
@@ -59,17 +59,18 @@ export class ButtonExample {
                 <limel-switch
                     label={control.label}
                     value={!!this.props[control.property]}
-                    onChange={event => { // tslint:disable-line:jsx-no-lambda prettier
-                        this.props = {
-                            ...this.props,
-                            ...{
-                                [control.property]:
-                                    (event.detail && control.value) || null,
-                            },
-                        };
-                    }}
+                    onChange={this.handleChange}
                 />
             );
         });
     }
+
+    private handleChange = (control) => (event: CustomEvent) => {
```

Comment:
The double arrow functions don't look right, I'm guessing that's why the button doesn't update anymore.